board/app/controllers/comment_controller.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+class CommentController extends AppController {
+
+const WRITE_COMMENT = 'write';
+const WRITE_COMMENT_END = 'write_end';
+const MAX_USER_IN_NEWSFEED = 1;
+
+    public function write() 
+    {
+        $thread_id = Param::get('thread_id');
+        $thread = Thread::get($thread_id);
+        $comment = new Comment();
```

Comment:
removed the declaration po :bow: 
--> https://github.com/maragatchalian/FoodFacts/commit/62f659ba58aa05520da674298ee0e57a25f9a72b
thank you :blush: